model/src/main/java/com/microsoft/java/bs/gradle/model/GradleProjectDependency.java

```diff
@@ -10,4 +10,6 @@
  */
 public interface GradleProjectDependency extends Serializable {
```

Comment:
The reason that I found the name may need to be change is because, we have some tests like `testGetGradleDependenciesWithTestToMain()`. This test tests that the main source set depends on test source set. But, both two source set belongs to the same project.

This is different from the definition of `project dependency` from Gradle document. I'm not sure if it's still proper to call it a project dependency.

Reply:
I can change the name.  How about `BuildTargetDependency`?  All the `XXXSourceSet` named classes could be changed to `XXXBuildTarget`?

Reply:
`BuildTargetDependency` looks good. And we can also update the javadoc explaining that it could be a project dependency or source set dependency.